docs/docsite/rst/dev_guide/developing_collections_creating.rst

```diff
@@ -81,6 +81,20 @@ To learn more about the ``ansible-galaxy`` command-line tool, see the :ref:`ansi
 
 .. _creating_collection_skeletons:
```

Comment:
This reference doesn't correspond to the title. Also, I wouldn't stick ADT in from of the collection template section. It's probably less controversial to put it after that section.